docs/Generators/TestChipIP.rst

```diff
@@ -60,6 +61,16 @@ the select signal once TileLink messages have begun sending.
 For an example of how to use the switcher, take a look at the ``SwitcherTest``
 unit test in the `Test Chip IP unit tests <https://github.com/ucb-bar/testchipip/blob/master/src/main/scala/Unittests.scala>`_.
 
+TileLink Ring Network
+---------------------
+
+TestChipIP provides a TLRingNetwork generator that has a similar interface
+to the TLXbar provided by RocketChip, but uses ring networks internally rather
+than crossbars. This can be useful for chips with very wide TileLink networks
+(many cores and L2 banks) that can sacrifice cross-section bandwidth to relieve
+congestion. Documentation on how to use the ring network can be found in
```

Comment:
Minor nit: Maybe explicitly state _wire_ congestion?  I can't imagine reducing cross-sectional bandwidth being favorable for network congestion.